#goal definition
std_msgs/Empty goal
---
# Define the result
bool execution_success
---
# Define a feedback message
float64 plan_success
